<!DOCTYPE html>
<html><head><title>UpdateDatasourceCredential</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">3 references to UpdateDatasourceCredential</div><div class="rA">Azure.AI.MetricsAdvisor.Tests (3)</div><div class="rG" id="Azure.AI.MetricsAdvisor.Tests"><div class="rF"><div class="rN">MetricsAdvisorAdministrationClient\DatasourceCredentialTests.cs (3)</div>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DatasourceCredentialTests.cs.html#51"><b>51</b>Assert.That(() =&gt; adminClient.<i>UpdateDatasourceCredential</i>(null), Throws.InstanceOf&lt;ArgumentNullException&gt;());</a>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DatasourceCredentialTests.cs.html#56"><b>56</b>Assert.That(() =&gt; adminClient.<i>UpdateDatasourceCredential</i>(credentialWithNullId), Throws.InstanceOf&lt;ArgumentNullException&gt;());</a>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DatasourceCredentialTests.cs.html#70"><b>70</b>Assert.That(() =&gt; adminClient.<i>UpdateDatasourceCredential</i>(credential, cancellationSource.Token), Throws.InstanceOf&lt;OperationCanceledException&gt;());</a>
</div>
</div>
</body></html>